DATA/UNIVERSE/SYSTEMS/IW02/iw02.ini

```diff
@@ -2097,7 +2098,7 @@ nickname = ZONE_Iw02_Sun_death
 pos = -4, 0, -13
 shape = SPHERE
 size = 5000
-damage = 10000
```

Comment:
Is there a particular reason why you increased the damage all the way to 250000? I just inspected many system files, and found that almost all sun death zones have the damage set to 10000 at most.

Reply:
I didn't do my due diligence and assumed the 250k damage of sun death zones was the default, especially since I knew 2Mil damage as the default for planet death zones.

I just did a full investigation of sun death zones for another mod and identified further simple errors in death zones (e.g. Br02's 1000 damage and Br04's issues), so I'll be doing another pass on those shortly.